agenta-web/src/components/Playground/Views/TestView.tsx

```diff
@@ -174,10 +181,21 @@ const App: React.FC<TestViewProps> = ({inputParams, optParams, URIPath}) => {
 
     const handleAddRow = () => {
         setTestList([...testList, {}])
+        setResultsList([...testList, {}].map(() => ""))
```

Comment:
no need to spread (...) when you're already doing map.

Also, a more readable way could be `Array(testList.length + 1).fill("")`

Reply:
@MohammedMaaz this was removed completely as the boxes need to keep their result state!